README.adoc

```diff
@@ -145,16 +183,33 @@ the custom resources created by this operator. Default value is `quay.io/integre
 
 === Metrics
 
+=== Monitoring Service (Metrics)
+
 The application-monitoring stack provisioned by the
-{application_monitoring_operator} can be used to gather metrics from
-the operator here.  Once you have provisioned that (or the
-ServiceMonitor CRD at a minimum), you can run the following commands
-to configure it:
+https://github.com/integr8ly/application-monitoring-operator[application-monitoring-operator] on https://github.com/integr8ly[Integr8ly]
```

Comment:
out of curiosity, why is the `{application_monitoring_operator}` link macro not being used here (like it was used in the removed text)?

Reply:
To impl the monitor we will use it.